io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/TypeSelectionPage.java

```diff
@@ -202,11 +202,11 @@ public boolean updateConfiguration(InstanceReader provider) {
 				Value.of(skipNlinesSpinner.getSelection()));
 
 		if (customFormat.isVisible()) {
```

Comment:
It seems to be not possible to not specify a value in the UI, but I think at least for CSV it should be the default to not specify the parameter because here the mechanism only works on guessing which values might be dates and the data may be modified unintentionally.

Also I think it would be good to make the description of the parameter more detailed for what the behavior is for Excel and CSV.

Suggestions:

Excel: "Format for imported date values (values are represented as strings)"

CSV: "Reformatting of dates (date values are tried to be detected automatically)"